lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -226,16 +200,20 @@
     p+p, p+ul, p+ol,
```

Comment:
A lot of these sibling behaviours will be defined in GOV.UK Frontend and inherited through the use of `@extend` – so a lot of these are either redundant or overriding things where they probably shouldn't be. Will need to review them to work out which make sense.

https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/core/_typography.scss#L161-L189